tests/test_tokenization_common.py

```diff
@@ -712,3 +733,147 @@ def _check_no_pad_token_padding(self, tokenizer, sequences):
 
             # add pad_token_id to pass subsequent tests
             tokenizer.add_special_tokens({"pad_token": "<PAD>"})
+
+    @require_torch
+    def test_torch_encode_plus_sent_to_model(self):
+        from transformers import MODEL_MAPPING, TOKENIZER_MAPPING
+
+        MODEL_TOKENIZER_MAPPING = merge_model_tokenizer_mappings(MODEL_MAPPING, TOKENIZER_MAPPING)
+
+        tokenizer = self.get_tokenizer()
+
+        if tokenizer.__class__ not in MODEL_TOKENIZER_MAPPING:
+            return
+
+        config_class, model_class = MODEL_TOKENIZER_MAPPING[tokenizer.__class__]
+        config = config_class()
```

Comment:
At first I wanted to initialize a configuration with only the length of the tokenizer to save memory, but doing so failed for some models which required additional configuration tweaking (like TransfoXL which required the cutoffs to be specified). This approach is the easiest in my opinion.

Reply:
Good for me!